docs/en/setup/backend/backend-vm-monitoring.md

```diff
@@ -3,38 +3,63 @@ SkyWalking leverages Prometheus node-exporter to collect metrics data from the V
 [OpenTelemetry receiver](opentelemetry-receiver.md) and into the [Meter System](./../../concepts-and-designs/meter.md).
 VM entity as a `Service` in OAP and on the `Layer: OS_LINUX`.
 
+SkyWalking also provides InfluxDB Telegraf to receive VMs' metrics data by [Telegraf receiver](./telegraf-receiver.md).
+The telegraf receiver plugin receiver, process and convert the metrics, then it send converted metrics to [Meter System](./../../concepts-and-designs/meter.md).
+VM entity as a `Service` in OAP and on the `Layer: OS_LINUX`.
+
 ## Data flow
+**For OpenTelemetry receiver:**
 1. The Prometheus node-exporter collects metrics data from the VMs.
 2. The OpenTelemetry Collector fetches metrics from node-exporter via Prometheus Receiver and pushes metrics to the SkyWalking OAP Server via the OpenCensus gRPC Exporter or OpenTelemetry gRPC exporter.
 3. The SkyWalking OAP Server parses the expression with [MAL](../../concepts-and-designs/mal.md) to filter/calculate/aggregate and store the results.
 
+**For Telegraf receiver:**
+1. The InfluxDB Telegraf [input plugins](https://docs.influxdata.com/telegraf/v1.24/plugins/) collects various metrics data from the VMs.
+2. The InfluxDB Telegraf send `JSON` format metrics by `HTTP` messages to Telegraf Receiver, then pushes converted metrics to the SkyWalking OAP Server [Meter System](./../../concepts-and-designs/meter.md).
+3. The SkyWalking OAP Server parses the expression with [MAL](../../concepts-and-designs/mal.md) to filter/calculate/aggregate and store the results.
 
 ## Setup
-
+**For OpenTelemetry receiver:**
 1. Setup [Prometheus node-exporter](https://prometheus.io/docs/guides/node-exporter/).
 2. Setup [OpenTelemetry Collector ](https://opentelemetry.io/docs/collector/). This is an example for OpenTelemetry Collector configuration [otel-collector-config.yaml](../../../../test/e2e-v2/cases/vm/prometheus-node-exporter/otel-collector-config.yaml).
 3. Config SkyWalking [OpenTelemetry receiver](opentelemetry-receiver.md).
 
+**For Telegraf receiver:**
+1. Setup InfluxDB Telegraf's `telegraf.conf file` according to [Telegraf office document](https://docs.influxdata.com/telegraf/v1.24/).
+2. Setup InfluxDB Telegraf's `telegraf.conf file` specific rules according to [Telegraf receiver document](telegraf-receiver.md).
+3. Config SkyWalking [Telegraf receiver](telegraf-receiver.md).
+
 ## Supported Metrics
+**For OpenTelemetry receiver:**
```

Comment:
What does this mean? OpenTelemetry receiver and Telegraf receiver don't share the metrics?

Reply:
If they are same, keep dataflow doc is fine, but you don't have to list two sheets for the same metrics. Right?

Reply:
> If they are same, keep dataflow doc is fine, but you don't have to list two sheets for the same metrics. Right?

But the metrics name and data source are different. If the telegraf receiver sheet is redundant I can delete it.

Reply:
Metrics name for UI, or metrics name from telegraf?

Reply:
> Metrics name for UI, or metrics name from telegraf?

Both metrics names are different. For example, telegraf metrics' `mem_total` shows `mem_total_info` in UI, but OpenTelemetry metrics' `meter_vm_memory_total` shows `memory_total` in UI.